lib/perl/Genome/Utility/Test.pm

```diff
@@ -312,10 +323,27 @@ sub data_dir_ok {
     return $data_dir;
 }
 
+sub get_test_dir {
```

Comment:
My issue with data_dir_ok is mostly academic, but I don't like the idea of using an 'ok' test for things that aren't actually testing the code we're trying to test.  A 'use_ok' is borderline but I'm alright with it since it actually loads the module we're trying to test.  I prefer to use the `note` function for things like this.